addons/desec.sh

```diff
@@ -81,6 +81,7 @@ do
         fi
     else
        break
+       msg_box "$DEDYNDOMAIN was successfully setup with deSEC! Now please continue with the DDNS and TLS setup for the domain."
```

Comment:
@enoch85 this msg_box will most likely never be shown because it is after the break keyword. (to get shown it must be before the keyword)

Reply:
Also, I think the word `setup` isn't correct here because it is a verb. And the verb for `setup` is `to set up`, afaik.

Reply:
actually, I was tired yesterday. ofc it should be after the break.

https://www.youtube.com/watch?v=eDMv3IHGpnA